.travis.yml

```diff
@@ -20,6 +20,7 @@ env:
   - SCRIPT="./dev-docker ./manage.py test"
   - SCRIPT="./scripts/test-docker"
   - SCRIPT="./manage.py compilemessages" INSTALLPIP=yes
+  - SCRIPT="docker run -v $(pwd):/squad --workdir /squad/squad fsfe/reuse reuse lint"
```

Comment:
why `/squad/squad`? with this you are running from inside the `squad` subdirectory, instead of from the root of the repository.

Reply:
I was just having a chat with @chaws right now about it ..we thought /squad is where main dir under the docker container and then /squad/squad is where the source files are. We were having doubts and not sure as well..so It should be 
docker run -v $(pwd):/squad --workdir /squad/ fsfe/reuse reuse lint?

Reply:
@terceiro should we run reuse on non-source files as well?

Reply:
It should be run in top level dir. We have scripts and tests dirs which are also part of SQUAD.

Reply:
ack

Reply:
@mwasilew exactly yes.